packages/core-ethereum/src/ethereum.mock.ts

```diff
@@ -0,0 +1,7 @@
+export const sampleChainOptions = {
```

Comment:
What about adding a type here, e.g. `sampleChainOption: ChainOptions = { ... }`? If we change the options then Typescript will complain and we won't miss the change

Reply:
Sure, valid suggestion

Reply:
Completed in 524f3b8